### DISCUSSION TITLE
[Feature Request] Diselect or reverse select?

### DISCUSSION THREAD

**Discussion by wyxgoishin** [Tyrrrz/YoutubeDownloader#320] [Q&A]
Upon donwloading playlist, I want to diselect several videos that I have already downloaded manually. But I only found that ctrl+A can select all and click can select one. Though current settings contain "skip existing files", but my existing files locate in several different folders which can be cumbersome to move to place where this software stores downloading files. So I wonder if there is an option for diselect or  shortcut for reverse select. Looking forward to your sincere reply, thank!

**Comment by Tyrrrz** [comment]
You can select/deselect individual items by holding Ctrl
